src/Request/Request.php

```diff
@@ -374,6 +376,8 @@ private function getDefaultHeaders()
         return [
             'User-Agent' => $this->userAgent(),
             'Accept' => 'application/json',
+            'X-Pantheon-Trace-Id' => $GLOBALS['PANTHEON_TRACE_ID'],
+            'X-Pantheon-Terminus-Command' => $this->terminusCommand(),
```

Comment:
@svanderbleek We think we see what you're going for here but have questions/concerns.  There are terminus plugins that implement sensitive commands like the (older) secrets and (newer) secrets manager plugin.  It would be infeasible to cover all sensitive scenarios with a block list.  

Is there a real need to have all of the command arguments in a request header, or would it be enough to just send which command was used in all cases?